### PR TITLE
course add/form changes

### PR DESCRIPTION
**Pull request by ninaprabhu**

### CODE REVIEW

**Comment by Tetheta** [comment]
Starting review

File: src/main/java/com/google/sps/servlets/SearchServlet.java [hunk 1]
```diff
@@ -223,24 +223,49 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+    List<Filter> filters = new ArrayList<>();
+    filters.add(new FilterPredicate("search-name", FilterOperator.EQUAL, name.toLowerCase()));
+    filters.add(new FilterPredicate("search-professor", FilterOperator.EQUAL, prof.toLowerCase()));
+    filters.add(new FilterPredicate("units", FilterOperator.EQUAL, units));
+    filters.add(new FilterPredicate("term", FilterOperator.EQUAL, term));
+    filters.add(new FilterPredicate("school", FilterOperator.EQUAL, school));
+
+    Query courseQuery = new Query("Course-Info");
+    if (!filters.isEmpty()) {
+      if (filters.size() == 1) {
```
Comment (Tetheta) [inline src/main/java/com/google/sps/servlets/SearchServlet.java:261]: When would filters be empty or size of only 1? Aren't you adding 5 filters directly above here?

File: src/main/webapp/search-style.css [hunk 2]
```diff
@@ -26,7 +26,7 @@ textarea, select, fieldset {
   width: 50%;
 }
 
-#search-button, #logout-button {
+#search-button, #logout-button, #private-logout-button, #add-to-search, #search-to-add, #add-button {
```
Comment (Tetheta) [inline src/main/webapp/search-style.css:29]: Might think about extracting this to a class name in the future if it further expands.

Comment (ninaprabhu) [reply src/main/webapp/search-style.css:29]: Made all of this into the class course-button

File: src/main/java/com/google/sps/servlets/SearchServlet.java (hunk 1 above)
Comment (ninaprabhu) [reply src/main/java/com/google/sps/servlets/SearchServlet.java:261]: good point, I think that was just a reflex stuck from the search in doGet

**Pull request merged**